tsconfig.json

```diff
@@ -8,5 +8,6 @@
     "sourceMap": true,
     "experimentalDecorators":true,
     "allowJs": true,
+    "resolveJsonModule": true,
```

Comment:
Where is this used?

Reply:
I use it to get the config value from our `package.json` and the electron version from its `package.json` in the unit test.